services/r2ctl/config/r2ctl.go

```diff
@@ -195,3 +199,11 @@ type policiesConfiguration struct {
 	StoragePolicies  []policy.StoragePolicy   `yaml:"storagePolicies"`
 	AggregationTypes []policy.AggregationType `yaml:"aggregationTypes"`
 }
+
+func decodeRuneInString(s string) []rune {
```

Comment:
nit: `toRunes` might be a better name. Sorry about going back and forth on this.